src/quantum/tp6-32bit.js

```diff
@@ -0,0 +1,89 @@
+import _ from 'lodash';
```

Comment:
Could you please do specific imports rather than `_`? (e.g. `import { chain, zipObject } from 'lodash;`)
It keeps the bundle size smaller.

Reply:
I do not think we can chain when if we import separate functions.  I will try something else

Reply:
resolving this conversation of more of the same with @helfi92